distrobox-create

```diff
@@ -219,6 +221,13 @@ while :; do
 				shift
 			fi
 			;;
+		--pre-init-hooks)
+			if [ -n "$2" ]; then
+				container_pre_init_hook="--pre-init-hooks ${2}"
```

Comment:
We need to put escaped quotes around the ${2}, or it will fail:

```
				container_pre_init_hook="--pre-init-hooks \"${2}\""
```

Will fix this

Reply:
fixed! Sorry for forgetting to re-test after refactoring